macOS redis5.0 make redisbloom ,the error is conflicting types for 'SB_NewChain'

/usr/local/Cellar/redis/5.0.8/module/RedisBloom/src/sb.c:95:10: error: conflicting types for 'SB_NewChain'
SBChain *SB_NewChain(uint64_t initsize, double error_rate, unsigned options, unsigned growth) {
         ^
/usr/local/Cellar/redis/5.0.8/module/RedisBloom/src/sb.h:34:10: note: previous declaration is here
SBChain *SB_NewChain(size_t initsize, double error_rate, unsigned options, unsigned growth);
         ^
1 error generated.
make: *** [/usr/local/Cellar/redis/5.0.8/module/RedisBloom/src/sb.o] Error 1

--- a/src/sb.h
+++ b/src/sb.h
@@ -31,7 +31,7 @@ typedef struct SBChain {
  *
  * Free with SBChain_Free when done.
  */
-SBChain *SB_NewChain(size_t initsize, double error_rate, unsigned options, unsigned growth);
+SBChain *SB_NewChain(uint64_t initsize, double error_rate, unsigned options, unsigned growth);
 
 /**
  * Create a new chain from a 'template'. This template will copy an existing

